Makefile

```diff
@@ -40,6 +40,8 @@ build: ## Builds the all docker setup.
 	docker compose build
 
 run-devserver: ## Runs the dev-server
+# Explicit creation sets proper ownership pre-container start before.
+	mkdir -p node_modules
 	docker compose up angular-build -d
 	$(MAKE) update.package
 	docker cp oppia-angular-build:/app/oppia/node_modules .
```

Comment:
Can you open a new issue and leave a comment here indicating that we have to find a more efficient method for connecting the folders rather than resorting to copying?

Reply:
sure, will do